src/flamegpu/visualiser/ModelVis.cpp

```diff
@@ -126,7 +142,12 @@ void ModelVis::updateBuffers(const unsigned int &sc) {
         visualiser->releaseMutex();
     }
 }
-
+void ModelVis::updateRandomSeed() {
+    if (visualiser) {
+        // Yolo thread safety, shouldn't matter if random seed is printed wrong for a single frame
```

Comment:
:vomiting_face: 

(I agree with the sentiment in the interest of time, but even still we should atleast make an issue so we can point at it if needed).